src/githubHelper.ts

```diff
@@ -146,20 +147,21 @@ export async function generatePRComment(testRunUrlsMap: TestRunUrlsMap): Promise
   try {
     pullRequestNumber = await getPullRequestNumber();
   } catch (error: any) {
-    core.error(`Error getting pull request number`);
-    core.error(error);
+    core.debug(`Got following error in getting pull request number`);
+    core.debug(error);
   }
 
   if (!pullRequestNumber) {
-    core.debug('Pull request number not found skipping comment creation');
+    core.warning('Unable to get pull request number for the commit, skipping comment creation')
```

Comment:
Should this be `info` instead? Running a k6 test after merge or as a periodic CI job may be a common scenario.

Reply:
Makes sense. Updating it